src/test/java/ldbc/snb/datagen/util/formatter/LongDateFormatterRegressionTest.java

```diff
@@ -0,0 +1,33 @@
+package ldbc.snb.datagen.util.formatter;
+
+import org.junit.Test;
+
+import java.util.Random;
+
+import static org.junit.Assert.assertTrue;
+
+public class LongDateFormatterRegressionTest {
```

Comment:
I can remove this test and the associated class prior to merging.

Reply:
Agreed - just drop this and `OldLongDateFormatter`.